pages/[address].tsx

```diff
@@ -228,7 +261,15 @@ const SynthPage: React.FC<Props> = ({ data, relatedSynths, change24h }) => {
               </ul>
             </>
           ) : (
-            <div> form here</div>
+            <LSP
+              data={data}
```

Comment:
Have you inspected this object? I think it has all the data you are fetching from contracs (other than balances)

Reply:
+1, seems either this or a lot of the props can be removed as they are redundant

Reply:
Pretty sure this is just the synth object. Or mostly.

It over injects info a bit, but it saves some complexity in the code. I can break down individual props I need *or* create a more catered object.
<img width="549" alt="Screen Shot 2021-08-04 at 10 35 17 AM" src="https://user-images.githubusercontent.com/12792146/128227743-6bda0946-b80c-4118-8b7d-bc3bfdfb54fc.png">